docs/schemas/ScenarioAndCoverageSets.schema.json

```diff
@@ -9,7 +9,6 @@
 	"additionalProperties": false,
 	"required": [
 		"touchstone_version",
-		"scenario",
-		"coverage_sets"
+		"scenario"
```

Comment:
It looks to me like below you always return the coverage sets, you just return them as null if the user doesn't have permission. So rather than making this not required, it should just be nullable?

Reply:
Ah, I found the webmodels PR. Never mind.